Wiki/Modules/FvwmBacker/index.md

```diff
@@ -3,14 +3,95 @@ title: FvwmBacker
 type: module
 weight: 430
 description: |
-  This module will change the backgound/wallpaper when changing desktops.
+  This module will change the background/wallpaper when changing desktops.
   Any command can be executed when changing desktops so this module can
   be used for more than just switching the background.
 ---
 # FvwmBacker
 
-The FvwmBacker module provides functionality to change the  background when
-changing  desktops.  Any command can be executed to change the backgrounds.
-Actually, any arbitrary command can be sent to fvwm to execute, so  you
+* TOC
+{:toc}
+
+The FvwmBacker module provides functionality to change the background when
+changing  desktops. Any command can be executed to change the backgrounds.
+Actually, any arbitrary command can be sent to fvwm to execute, so you
 could also do things such as changing window border colors, etc.
 
+Between the desktops you can personalize them by giving them individual
+background colors.
+
+{% fvwm2rc %}
+*FvwmBacker: Command (Desk 0) -solid white
+*FvwmBacker: Command (Desk 2) Colorset 5
+{% endfvwm2rc %}
+
+Instead of colors, you can define background images:
+
+{% fvwm2rc %}
+*FvwmBacker: Command (Desk 0) Exec exec fvwm-root $HOME/background-mono.png
+*FvwmBacker: Command (Desk 2) Exec exec fvwm-root $HOME/wallpaper-nature.jpg
+{% endfvwm2rc %}
+
+Also to change the background for each page:
+
+{% fvwm2rc %}
+*FvwmBacker: Command (Desk 0, Page 0 1) Colorset 5
+*FvwmBacker: Command (Desk 2, Page 1 1) $HOME/wallpaper-nature.jpg
+{% endfvwm2rc %}
```

Comment:
This example wouldn't work, first you need to run some program to set the background, such `Exec exec fvwm-root`. And if you are going to use `$HOME`, at least use `$[HOME]`.